### ISSUE TITLE
M6 Live TV not playing (Kodi 18.7)

### ISSUE THREAD

**Issue by ludovic-fletanchard** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#471]
### Description
When I try to play M6 in Live TV,  the Kodi loading overlay stays for 30sec approximately, and then disappear and I'm still on the channels menu

### To Reproduce
Steps to reproduce the behaviour:
1. Live TV > France > M6

### Expected behaviour
It should play M6 :)

### Current behaviour
It doesn't play M6, doesn't show any error message, just stays on the channel selection menu.

### Context:
- **Hardware**: Pi4
- **OS**: Raspbian 10 (Buster)
- **Kodi version**: 18.7
- **Catch-up TV & More version**: 0.2.32
- **Installation method**: Repository

### Additional context
My 6play credentials are properly set, the account works on other devices.

### Debug log
```
2021-03-13 16:16:23.762 T:2972821536   ERROR: Control 55 in window 10025 has been asked to focus, but it can't
2021-03-13 16:17:04.500 T:2076188880  NOTICE: [script.module.inputstreamhelper] Widevine update check was made on 2021-03-13 12:50
2021-03-13 16:17:05.605 T:2972821536  NOTICE: VideoPlayer::OpenFile: plugin://plugin.video.catchuptvandmore/resources/lib/channels/fr/6play/get_live_url/?_pickle_=80027d71012858070000006974656d5f6964710258020000006d36710355075f7469746c655f7104581a0000004d3620e2809420436861737365757273206427617070617274275808000000786d6c74765f696471055814000000433131382e6170692e74656c6572616d612e66727106752e
2021-03-13 16:17:05.691 T:2043179216  NOTICE: Creating InputStream
2021-03-13 16:17:06.830 T:2043179216  NOTICE: Creating Demuxer
2021-03-13 16:17:06.831 T:2043179216  NOTICE: Opening stream: 1001 source: 256
2021-03-13 16:17:06.853 T:2043179216  NOTICE: Creating video codec with codec id: 27
2021-03-13 16:17:06.866 T:2043179216  NOTICE: Creating video thread
2021-03-13 16:17:06.866 T:2034786512  NOTICE: running thread: video_thread
2021-03-13 16:17:06.866 T:2043179216  NOTICE: Opening stream: 1002 source: 256
2021-03-13 16:17:06.900 T:2043179216  NOTICE: Finding audio codec for: 86018
2021-03-13 16:17:06.900 T:2043179216  NOTICE: CDVDAudioCodecFFmpeg::Open() Successful opened audio decoder aac
2021-03-13 16:17:06.901 T:2043179216  NOTICE: Creating audio thread
2021-03-13 16:17:06.901 T:1992822992  NOTICE: running thread: CVideoPlayerAudio::Process()
2021-03-13 16:17:07.042 T:2026393808   ERROR: AddOnLog: InputStream Adaptive: Download https://live.quortex.bedrock.tech/6play/short/enc/m6/noscte/videohd-1280x720/segment_338915.m4s failed with error: 404
2021-03-13 16:17:17.415 T:2001215696   ERROR: Previous line repeats 10 times.
2021-03-13 16:17:17.415 T:2001215696   ERROR: AddOnLog: InputStream Adaptive: Download https://live.quortex.bedrock.tech/6play/short/enc/m6/noscte/audio-fre-96000/segment_338915.m4s failed with error: 404
2021-03-13 16:17:27.638 T:2043179216   ERROR: Previous line repeats 10 times.
2021-03-13 16:17:27.638 T:2043179216  NOTICE: CVideoPlayer::OnExit()
2021-03-13 16:17:27.638 T:2043179216  NOTICE: VideoPlayer: eof, waiting for queues to empty
2021-03-13 16:17:27.639 T:2043179216  NOTICE: Closing stream player 1
2021-03-13 16:17:27.639 T:2043179216  NOTICE: Waiting for audio thread to exit
2021-03-13 16:17:27.641 T:1992822992  NOTICE: thread end: CVideoPlayerAudio::OnExit()
2021-03-13 16:17:27.641 T:2043179216  NOTICE: Closing audio device
2021-03-13 16:17:27.641 T:2043179216  NOTICE: Deleting audio codec
2021-03-13 16:17:27.641 T:2043179216  NOTICE: Closing stream player 2
2021-03-13 16:17:27.641 T:2043179216  NOTICE: waiting for video thread to exit
2021-03-13 16:17:27.641 T:2034786512   ERROR: Got MSGQ_ABORT or MSGO_IS_ERROR return true
2021-03-13 16:17:27.641 T:2034786512  NOTICE: thread end: video_thread
2021-03-13 16:17:27.642 T:2043179216  NOTICE: deleting video codec
2021-03-13 16:17:27.729 T:2972821536  NOTICE: CVideoPlayer::CloseFile()
2021-03-13 16:17:27.729 T:2972821536  NOTICE: VideoPlayer: waiting for threads to exit
2021-03-13 16:17:27.729 T:2972821536  NOTICE: VideoPlayer: finished waiting
2021-03-13 16:17:27.729 T:2972821536  NOTICE: CVideoPlayer::CloseFile()
2021-03-13 16:17:27.730 T:2972821536  NOTICE: VideoPlayer: waiting for threads to exit
2021-03-13 16:17:27.730 T:2972821536  NOTICE: VideoPlayer: finished waiting
```

**Comment by wwark** [comment]
Hi,
Your issue is follow in this ticket https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/issues/447
Thanks.

**Comment by ludovic-fletanchard** [comment]
Hello,

I'm definitely using CodeQuick **1.0.2**, CUTV **0.2.33**, but I am still getting the exact same error. Am I missing something, or is it definitely the same issue ?

Thanks a lot

**Comment by ludovic-fletanchard** [comment]
```
ERROR: AddOnLog: InputStream Adaptive: Download https://live.quortex.bedrock.tech/6play/short/enc/m6/noscte/videohd-1280x720/segment_439143.m4s failed with error: 404
ERROR: Previous line repeats 10 times.
ERROR: AddOnLog: InputStream Adaptive: Download https://live.quortex.bedrock.tech/6play/short/enc/m6/noscte/audio-fre-96000/segment_439143.m4s failed with error: 404
ERROR: Previous line repeats 10 times.
```
@wwark Are these lines related to the CodeQuick issue ?

**Comment by wwark** [comment]
hi @ludovic-fletanchard,

I see also you have an old version of Kodi 18. Could you please try to update to Kodi 18.9 (Last Version) ?
This one comes with the last version of Inputstream.Adaptive used to play DRM content.

Thanks,

**Comment by ludovic-fletanchard** [comment]
Oookay I'm starting to get it. I have the same issue as  in #300 I guess. But being on Raspbian, the latest version available is 18.7, so I'm stuck with the old `Inputstream.Adaptive`. I guess I simply have to wait.

Thank you very much for your time and patience, so I'm closing the ticket again and for good :)
